### PR TITLE
High-level interface for speed control of actuators

### PR DESCRIPTION
**Pull request by dogoepp**
This allows for easy an generic control of target speed for actuators both in joint and wheel mode. It has been tested for AX-12+, MX-64 and MX-28AT actuators.

Nonetheless, a review of the code by at least one of the teem members will be most appreciated.

### CODE REVIEW

File: src/dynamixel/servos/protocol_specific_pacquets.hpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#ifndef DYNAMIXEL_SERVOS_PROTOSPECIFICPACQUETS_HPP_
```
Comment (costashatz) [inline src/dynamixel/servos/protocol_specific_pacquets.hpp]: The name of the file shouldn't be `protocol_specific_packets`? And not `protocol_specific_pacquets`?

Comment (dogoepp) [reply src/dynamixel/servos/protocol_specific_pacquets.hpp]: thanks, solved.

**Comment by fedeallocati** [comment]
Hey, I think this looks really good, and the Protocol Specific Packets is a nice idea.

**Comment by dogoepp** [comment]
Hey Federico,
Thanks a lot for the feedback. I'm flattered by your remark :-)

So, **OK to merge** for you ?

What about you @costashatz ?

**Comment by costashatz** [comment]
> What about you @costashatz ?

If @fedeallocati says it is ok and you tested on actual hardware, it's ok for me too..

**Comment by dogoepp** [comment]
**merging**

**Comment by dogoepp** [comment]
Still, it would be nice if @jbmouret could check at least `src/dynamixel/servos/protocol_specific_packets.hpp` for general C++ soundness.

**Pull request merged**